game_data_scraper/game_data_scraper.py

```diff
@@ -58,30 +72,45 @@ def _save_response(self, response: scrapy.http.Response, response_id: int):
             s3_client.upload_file(
                 full_local_path,
                 S3_SCRAPER_BUCKET,
-                f"data_dirty/{save_subfolder}/{save_filename}",
+                f"data_dirty/{self.local_save_path}/{save_filename}",
             )
 
 
+def set_vars_depending_on_scraper_type(scraper_type):
+    json_urls_prefix = SCRAPER_CONFIG[scraper_type]["s3_location"]
+    scrapy_bot_name = SCRAPER_CONFIG[scraper_type]["bot_name"]
+    local_path = SCRAPER_CONFIG[scraper_type]["local_path"]
+    scraped_games_save = SCRAPER_CONFIG[scraper_type]["local_subfolder"]
+
+    return json_urls_prefix, scrapy_bot_name, local_path, scraped_games_save
+
+
 if __name__ == "__main__":
 
-    filename = sys.argv[1]
+    scraper_type = sys.argv[1]
+    filename = sys.argv[2]
+
+    json_urls_prefix, scrapy_bot_name, local_path, scraped_games_save = (
+        set_vars_depending_on_scraper_type(scraper_type)
+    )
+
     print(filename)
 
     # get file from local if dev, else from S3
 
     if ENV == "dev":
-        scraper_urls_raw = json.load(open(f"scraper_urls_raw/{filename}.json"))
+        scraper_urls_raw = json.load(open(f"{local_path}/{filename}.json"))
```

Comment:
Generally speaking I'm not a fan of if statements like this in code (what if we introduce a QA environment?).  For now let's leave it.  Other solution would be to just pass an `ENV` file, an `ENV` dataclass, or a data loader.